### DISCUSSION TITLE
What is the best charging limit for my use case?

### DISCUSSION THREAD

**Discussion by Emreler1** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1159] [Q&A] [answered]
Hi!
I have been using Aldente Pro probably since I bought my Macbook Air M1 2 years ago. It's battery health is as follows:
  Health Information:
  Cycle Count:	197
  Condition:	Normal
  Maximum Capacity:	%94

The notebook is kept in clamshell mode 5 days a week 24 hours a day and only used in normal notebook mode 2 days of the week. When in clamshell mode it is connected to a Caldigit USB docking station and charged from there. I do not if that is fast charging or not but I can see that the macbook is being charged at around 31 watts, so I assume it is not being fast charged.

Anyway, when I use it in clamshell mode, I have been keeping it at 50% with 2% sailing mode. But I also scheduled it to charge to 100% right before the 2 day normal full day mode and scheduled to revert back to 50% at the end of the 2nd day. I also scheduled it calibrate in every two weeks.

I was assuming this was a healthy method. But I am not sure. What do you think is better in the long term? My current method or using 80% limit all the time with 2% sailing?

Cheers

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @Emreler1,

In general, I want to mention that something like "the best charging limit" does not exist, we only have indications and rules of thumb from the literature.

In my opinion, you are already doing a good job of prolonging the lifespan of your battery and your Health Information is showing that.
Minor improvements could be made by changing your scheduled charge to 100% to a slightly lower limit between 80%-95% since charging and staying at 100% is quite harmful to batteries. However, if you need 100% battery regularly during these two days, of course, you should continue charging to 100%.
The charge limit of 50% is also good choice but will most likely lead to battery calibration issues. However, since you are running the Calibration Mode every two weeks this should not be an issue for you.

Best,
Matthias
